docs/specs/learn.yml

```diff
@@ -1575,3 +1575,45 @@ outputs:
         }
       }
     }
+---
+# Validate secure link for notebook url
+inputs:
+  docfx.yml: |
+    markdownValidationRules: rules.json
+  rules.json: |
+    {
+      "links": {
+        "rules": [
+          {
+            "type": "SecureLinks",
+            "message": "Link '{0}' is insecure. Links to Microsoft sites must use 'https' instead of 'http'.",
+            "code": "insecure-link",
+            "severity": "SUGGESTION",
+            "contentTypes": ["moduleunit"],
+            "domains": [
+              "microsoft.com",
+              "office.com",
+              "msdn.com",
+              "aka.ms",
+              "visualstudio.com",
+              "azure.com",
+              "technet.com",
+              "githubusercontent.com"
+            ]
+          }
+        ]
+      }
+    }
+  _themes/ContentTemplate/schemas/ModuleUnit.schema.json: |
+    {
+      "properties": {
+        "notebook" : {"type": "string", "contentType": "Href"}
+      }
+    }
+  a.yml: |
+    ### YamlMime:ModuleUnit
+    notebook: http://raw.githubusercontent.com/test.ipynb
```

Comment:
> notebook: http://raw.githubusercontent.com/test.ipynb [](http://example.com/codeflow?start=3&length=54)

Does docs-ui needs to be updated too?
[https://dev.azure.com/ceapex/Engineering/_git/docs-ui?path=%2Fpackages%2Fschemas%2Fsrc%2FModuleUnit.schema.json&version=GBmingwli%2Fexport-loc-props&line=24&lineEnd=25&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents](https://dev.azure.com/ceapex/Engineering/_git/docs-ui?path=%2Fpackages%2Fschemas%2Fsrc%2FModuleUnit.schema.json&version=GBmingwli%2Fexport-loc-props&line=24&lineEnd=25&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents)

it got no ContentType

Reply:
Correct, I will update the docs-ui schema later after this PR.